### PR TITLE
Disable ephemeral storage in Katib Config

### PR DESCRIPTION
**Pull request by andreyvelich**
Fixes: https://github.com/kubeflow/katib/issues/1358.
Blocked by: https://github.com/kubeflow/website/pull/2312.

I added info about disabling ephemeral storage for suggestion and metrics collector container.

/assign @gaocegege @johnugeorge @kylepad 
/cc @8bitmp3

### CODE REVIEW

**Comment by andreyvelich** [comment]
This PR is ready
/cc @gaocegege @johnugeorge @8bitmp3

File: content/en/docs/components/katib/katib-config.md [hunk 1]
```diff
@@ -85,6 +85,21 @@ a default value is set automatically.
    - `cpu = 500m`
    - `ephemeral-storage = 5Gi`
 
+   It is possible to not request the `ephemeral-storage` resource for the
+   metrics collector's container. For example to use
+   [GKE nodepool autoscalers](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations).
+   To do that, set the negative values for the `ephemeral-storage` requests and
+   limits in your Katib config:
```
Comment (8bitmp3) [inline content/en/docs/components/katib/katib-config.md:92]: The second sentence is not complete, I think.
The third sentence is not clear about what "that" is.
Can you revise them please?

File: content/en/docs/components/katib/katib-config.md [hunk 2]
```diff
@@ -205,6 +220,21 @@ a default value is set automatically.
    - `cpu = 500m`
    - `ephemeral-storage = 5Gi`
 
+   It is possible to not request the `ephemeral-storage` resource for the
+   suggestion's container. For example to use
+   [GKE nodepool autoscalers](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations).
+   To do that, set the negative values for the `ephemeral-storage` requests and
+   limits in your Katib config:
```
Comment (8bitmp3) [inline content/en/docs/components/katib/katib-config.md:227]: Same as https://github.com/kubeflow/website/pull/2315/files#r521391860

File: content/en/docs/components/katib/katib-config.md (hunk 1 above)
Comment (andreyvelich) [reply content/en/docs/components/katib/katib-config.md]: Sure, @8bitmp3.
What do you think about this:
```
 You can run your metrics collector container without requesting
 the `ephemeral-storage` resource from the Kubernetes cluster.
 For instance, if you want to use
 [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations)
 for your Katib experiments. To remove the `ephemeral-storage` resource from
 the metrics collector container, set the negative values for the
 `ephemeral-storage` requests and limits in your Katib config:
```

File: content/en/docs/components/katib/katib-config.md [hunk 3]
```diff
@@ -85,11 +85,13 @@ a default value is set automatically.
...
-   [GKE nodepool autoscalers](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations).
-   To do that, set the negative values for the `ephemeral-storage` requests and
-   limits in your Katib config:
+   You can run your metrics collector container without requesting
+   the `ephemeral-storage` resource from the Kubernetes cluster.
+   For instance, if you want to use
+   [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations)
+   for your Katib experiments. To remove the `ephemeral-storage` resource from
+   the metrics collector container, set the negative values for the
+   `ephemeral-storage` requests and limits in your Katib config:
```
Comment (8bitmp3) [inline content/en/docs/components/katib/katib-config.md:94]: This sentence looks like it starts with "if" and doesn't end.

Maybe you can try this, if I understood the logic correctly:

```suggestion
   For instance, when using the
   [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations)
   for your Katib experiments, you can remove the `ephemeral-storage` resource from
   the metrics collector container by setting the negative values for the
   `ephemeral-storage` requests and limits in your Katib config as follows:
```

WDYT?

File: content/en/docs/components/katib/katib-config.md [hunk 4]
```diff
@@ -220,11 +222,13 @@ a default value is set automatically.
...
-   [GKE nodepool autoscalers](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations).
-   To do that, set the negative values for the `ephemeral-storage` requests and
-   limits in your Katib config:
+   You can run your suggestion container without requesting
+   the `ephemeral-storage` resource from the Kubernetes cluster.
+   For instance, if you want to use
+   [Google Kubernetes Engine cluster autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#limitations)
+   for your Katib experiments. To remove the `ephemeral-storage` resource from
+   the suggestion container, set the negative values for the
+   `ephemeral-storage` requests and limits in your Katib config:
```
Comment (8bitmp3) [inline content/en/docs/components/katib/katib-config.md:231]: Same as the comment before - PTAL

**Review by andreyvelich — COMMENTED** [review]
I made changes.
PTAL @8bitmp3.

**Comment by 8bitmp3** [comment]
Thank you and please assign/ping the Katib approvers here 👍 I'm not sure I'm part of OWNERS here

**Comment by andreyvelich** [comment]
Thanks @8bitmp3.
/assign @gaocegege @johnugeorge 
/approve

**Pull request merged**